packages/graphqlsp/src/autoComplete.ts

```diff
@@ -8,7 +8,12 @@ import {
   getAutocompleteSuggestions,
   getTokenAtPosition,
   getTypeInfo,
+  RuleKinds,
+  State,
+  RuleKind,
+  CompletionItem,
 } from 'graphql-language-service';
+import { runOnlineParser } from 'graphql-language-service/dist/interface/getAutocompleteSuggestions';
```

Comment:
The onlineParser is exposed at the root, let's not import this specifically

Reply:
> The tests could be due to an error on my behalf with making graphql a dependency. It should most likely be a peerDep.
> 
> The existing test seems to indicate it works well. I still need to add some more tests to test variable types and argument completion

in this case I was trying to set up unit tests, will try to add my tests as e2e

Reply:
will change the online parser!